common/utils/polyvinyl.js

```diff
@@ -5,11 +5,11 @@ import castToObservable from '../app/utils/cast-to-observable.js';
 
 
 // createFileStream(
-//   files: Dictionary[Path, PolyVinyl]
+//   files: List[ PolyVinyl ]
```

Comment:
You don't need to include `List` here. To express an array in rtype its just `[]`. To express a typed array it's `[...Polyvinyl]`.